### DISCUSSION TITLE
Falcon Sensor Installation process using PowerShell

### DISCUSSION THREAD

**Discussion by 1omega** [CrowdStrike/psfalcon#110] [Q&A] [answered]
Show actual install process for the sensors. I see it stops at just the installers being downloaded.

**Comment by bk-cs** [comment]
This is something that would involve the creation of a script--the CrowdStrike APIs don't install the sensor for you, but they provide a way to download a sensor version.

There a couple downsides with using PSFalcon for the entire process:
1. The computer you want to install the Falcon Sensor on would need to have PSFalcon beforehand
2. The target computer would also need the script that runs through the appropriate PSFalcon commands

I've made a self-contained script used by some CrowdStrike customers that will check a Sensor Update policy (by name) for the version that should be installed (or get the latest version), download the sensor, install it, then clean up after itself while logging to the Windows Event Log.

Unfortunately it's a pretty complex script which isn't suitable for posting as an example a under [Basic Scripts](https://github.com/CrowdStrike/psfalcon/wiki/Basic-Scripts). If you get in touch with your CrowdStrike sales team, they should be able to get you a copy. I recommend going that route.

**Comment by bk-cs** [comment, marked as answer]
You can find an example Falcon installation script here: https://github.com/CrowdStrike/Cloud-AWS/blob/master/Agent-Install-Examples/powershell/sensor_install.ps1
